tests/NHSD.BuyingCatalogue.Identity.Api.UnitTests/Controllers/UsersControllerTests.cs

```diff
@@ -304,12 +307,52 @@ public async Task DisableUser_UserRepository_UpdateAsync_CalledOnce()
             var context = UsersControllerTestContext.Setup();
             using var controller = context.Controller;
 
-            context.User = ApplicationUserBuilder.Create().WithDisabled(false).BuildBuyer();
+            context.User = ApplicationUserBuilder.Create().WithDisabled(false).Build();
 
             await controller.DisableUserAsync(context.User.Id);
 
-            context.UsersRepositoryMock.Verify(x => x.GetUserByIdAsync(context.User.Id), Times.Once);
+            context.UsersRepositoryMock.Verify(x => x.GetByIdAsync(context.User.Id), Times.Once);
             context.UsersRepositoryMock.Verify(x => x.UpdateAsync(context.User), Times.Once);
         }
     }
+
+    internal sealed class UsersControllerTestContext
```

Comment:
I think this should live in its own file or alternatively be made a private nested class.

Reply:
It used to be in its own file. The worry here is others may want to use it for other tests or slightly change it to fit another need. Changed it to a private nested class as the context of it should be constrained to this test fixture. May lead to problems if start sharing test contexts.